tabbycat/tournaments/utils.py

```diff
@@ -199,7 +200,7 @@ def send_standings_emails(tournament, teams, request, round):
 
     for team in teams:
         context['POINTS'] = str(team.points_count)
-        context['TEAM'] = team.short_name
+        context['TEAM'] = get_team_short_name(team)
```

Comment:
Hmm, does this one need to be coded? This one's only sent to the team to whom the points count relates, right? (I'm open to discussion on this, I just assumed it would be fine with the real name.)

Reply:
Yes, it is only sent to members of the team.